ocs_ci/utility/utils.py

```diff
@@ -5468,3 +5468,38 @@ def wait_custom_resource_defenition_available(crd_name, timeout=600):
             dont_raise=True,
         )
     )
+
+
+def get_running_odf_version():
```

Comment:
Should this be moved to version module?
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/version.py#L113

Reply:
done